src/backend/el.d

```diff
@@ -65,22 +65,24 @@ struct elem
     enum IDelem = 0x4C45;   // 'EL'
     //elem_debug(e) assert((e)->id == IDelem)
 
+    version (OSX) // workaround https://issues.dlang.org/show_bug.cgi?id=16466
+        align(16) eve EV; // variants for each type of elem
+    else
+        eve EV;           // variants for each type of elem
+
```

Comment:
So here the old layout is: `debug short`, `ubyte`, `ubyte`, `eflags_t`, `eve`, `union` (pointer), pointer, `Srcpos`. The new layout is:  `debug short`, `eve`, `ubyte`, `ubyte`, `eflags_t`, `union`, pointer, `Srcpos`.

* First off the first `debug short` is most problematic and should most likely be moved to the end.

* The layout best starts with the most demanding alignment, probably `eve`.

* Then group large data together and small data together so as you fill the knapsack properly.

Something like: `eve`, pointer, `union`, `Srcpos`, `eflags_t`, `debug short`, `ubyte`, `ubyte`